transports/janus_websockets.c

```diff
@@ -618,12 +611,22 @@ int janus_websockets_init(janus_transport_callbacks *callback, const char *confi
 			item = janus_config_get(config, config_general, janus_config_type_item, "ws_ip");
 			if(item && item->value) {
 				ip = (char *)item->value;
+				if(inet_net_pton(AF_INET, ip, &addr, sizeof(addr))>0) {
+				wscinfo.options |= LWS_SERVER_OPTION_DISABLE_IPV6;
+				}
 				char *iface = janus_websockets_get_interface_name(ip);
 				if(iface == NULL) {
 					JANUS_LOG(LOG_WARN, "No interface associated with %s? Falling back to no interface...\n", ip);
 				}
-				ip = iface;
+				//ip = iface;
```

Comment:
Not sure this is okay.
The `ip` variable is re-used to store the interface name for that address and then set the `iface` member of `lws_context_creation_info`.
```c
info.iface = ip ? ip : interface;
```
According lo lws docs:
```
NULL to bind the listen socket to all interfaces, or the interface name, eg, "eth2"
```

Reply:
Yes, at some point i could not bind to a specific ipv4 ip, without this change, but I could not replicate that behaviour, so
I removed this change.